packages/vanilla/src/components/footer/README.md

```diff
@@ -0,0 +1,119 @@
+# Footer
+
+> The footer component is a required navigational pattern for IBM.com that
+> displays consistently at the bottom of each page.
+
+## Getting started
+
+Here's a quick example to get you started.
+
+```scss
+// yourapplication.scss
+@import '@carbon/type/scss/font-face/mono';
+@import '@carbon/type/scss/font-face/sans';
+@include carbon--font-face-mono();
+@include carbon--font-face-sans();
+
+import '@carbon/ibmdotcom-styles/scss/components/footer/index.scss';
+```
+
+> 💡 Only import font's once per usage
+
+```javascript
+import { Footer } from '@carbon/ibmdotcom-react';
+import 'yourapplication.scss';
+
+console.log(
+  footer({
+    type: '', // tall | short
+  })
+);
+```
+
+> 💡 Don't forget to import the footer styles from
+> [@carbon/ibmdotcom-styles](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/styles).
+
+### Initialize
+
+> Init example below
+
+```javascript
```

Comment:
I wouldn't print the init function here, but tell the user what the init function does (it initializes the accordion view for mobile). Then tell the user how to initialize it:

```
import { Footer, footerTemplate } from '@carbon/ibmdotcom-vanilla';

const content = footerTemplate({
  type: 'tall',
  footerMenu: { ...footer menu content ... }
  footerThin: { ... footer thin content ... }
}); // returns the markup
const elem = document.getElementById('yourFooterDiv');
elem.innerHTML = content; // sets the footer content into the element
Footer.init(elem); // initializes the footer
```